contrib/envoyproxy/go-control-plane/envoy.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2024 Datadog, Inc.
 
-package go_control_plane
+package gocontrolplane
```

Comment:
Let's make a final decision on whether we want to support this change in v2; if we don't, I don't think we would have a chance to change this ever again.

Reply:
I'm in favour of supporting this change in v2.